vlsi/Makefile

```diff
@@ -51,6 +51,11 @@ else
 	OBJ_DIR          ?= $(vlsi_dir)/$(VLSI_OBJ_DIR)/$(long_name)-$(TOP)
 endif
 
+ifneq ($(ASPECTS), )
```

Comment:
I would move this block of code to the `common.mk` file (since this is agnostic to running in a VLSI flow) and I would add a bit of help text on how to use this `ASPECTS` variable here: https://github.com/ucb-bar/chipyard/blob/27468942b36865920a86dfc964af667e8cc6940a/common.mk#L13-L23